src/ruuvitag-adapter.ts

```diff
@@ -157,6 +155,20 @@ export class RuuviTagAdapter extends Adapter {
     this.knownDevices = {};
     addonManager.addAdapter(this);
 
+    // @tim - the manifest that is loaded (I can't work out how) does not have the id
+    //        The id is required to load the configuration, so I hack it in here
+    //        Sometimes I've seen a generated manifest.json in the lib directory
+    manifest.id = manifest.id || 'ruuvitag-adapter';
+
+    const config = getDefaultConfig();
+    const db = new Database(manifest.id);
```

Comment:
The `manifest` predates the `manifest.json` file.
That's why the structure is based on the `package.json` file.
Unfortunately, there is no way to ever change it without breaking existing code.
Instead of `id` and `name` its `name` and `display_name`.
```suggestion
    const db = new Database(manifest.name);
```